cmd/snap-preseed/reset.go

```diff
@@ -60,6 +61,7 @@ func resetPreseededChroot(preseedChroot string) error {
 		filepath.Join(dirs.SnapUserServicesDir, "default.target.wants", "snap.*.service"),
 		filepath.Join(dirs.SnapUserServicesDir, "sockets.target.wants", "snap.*.socket"),
 		filepath.Join(dirs.SnapUserServicesDir, "timers.target.wants", "snap.*.timer"),
+		filepath.Join(runinhibit.InhibitDir, "*.lock"),
```

Comment:
Should this glob be more specific and just use `snapd.lock` ?

Reply:
no, there are many locks there called \<instance-name>.lock.  The test is just a bit misleading right now I think